src/components/Button/Button.tsx

```diff
@@ -46,6 +46,8 @@ export interface Props {
   monochrome?: boolean;
   /** Forces url to open in a new tab */
   external?: boolean;
+  /** Tells the browser to download the url instead of opening it. Value is the hint for the saved filename.. */
```

Comment:
```suggestion
  /** Tells the browser to download the url instead of opening it. Value is the hint for the saved filename. */
```

Reply:
I was expecting `Value` to be a property. Maybe it would be helpful to clarify that the string value is used as the hint?

Reply:
[We talked on slack, and we have a plan!]